GOVERNANCE.md

```diff
@@ -0,0 +1,87 @@
+# Project Governance
+
+The ModelPack project is dedicated to creating a universal, vendor-neutral standard for packaging, versioning, and running AI/ML projects that aligns with the OCI specification. The ModelPack is especially important for organizations who are self-hosting models (open source, or self-built) and want to be able to use the tools and processes they've proven with containers, to manage their AI/ML project lifecycle.
+
+This is not an implementation, it is the specification on which compliant implementations can be built.
+
+This governance document explains how the project is run.
+
+- [Values](#values)
+- [Maintainers](#maintainers)
+- [Becoming a Maintainer](#becoming-a-maintainer)
+- [Meetings](#meetings)
+- [Code of Conduct Enforcement](#code-of-conduct)
+- [Modifications](#modifying-this-charter)
+
+## Values
+
+The ModelPack project leadership embraces the following values:
+
+- **Openness**: Communication and decision-making happens in the open and is discoverable for future reference. As much as possible, all discussions and work take place in public forums and open repositories.
+
+- **Fairness**: All stakeholders have the opportunity to provide feedback and submit contributions, which will be considered on their merits.
+
+- **Community over Product or Company**: Sustaining and growing our community takes priority over shipping code or sponsors' organizational goals. Each contributor participates in the project as an individual.
+
+- **Inclusivity**: We innovate through different perspectives and skill sets, which can only be accomplished in a welcoming and respectful environment.
+
+- **Participation**: Responsibilities within the project are earned through participation, and there is a clear path up the contributor ladder into leadership positions.
+
+## Maintainers
+
+ModelPack maintainers have write access to the [project GitHub repository](https://github.com/modelpack/model-spec). They can merge patches and are expected to review large or critical patches personally. The current maintainers can be found in [MAINTAINERS.md](./MAINTAINERS.md). Maintainers collectively manage the project's resources and contributors.
```

Comment:
There is a discussion in the slack channel about how maintainers with the project should be organized. Ideally, project maintainers have access to the core repositories including:

* community
* model-spec
* modctl

Additional supplementary repositories can include a separate list of maintainers (such as for tools etc) which are not primary org maintainers